santoku/aws/s3_handler.py

```diff
@@ -308,8 +351,8 @@ def generate_quicksight_manifest(
         """
         Generates a QS manifest JSON file.
 
-        Generates a QS manifest JSON file from a list of `S3 URIs` and/or `S3 URI prefixes` and upload them
-        to the `bucket`.
+        Generates a QS manifest JSON file from a list of `S3 URIs` and/or `S3 URI prefixes` and

```

Comment:
Generates a QS manifest JSON file from a list of `S3 URIs` and/or `S3 URI prefixes` that it is uploaded to `bucket`